odl/tomo/geometry/parallel.py

```diff
@@ -426,6 +427,63 @@ def __repr__(self):
     rotation_matrix = AxisOrientedGeometry.rotation_matrix
 
 
+def parallel_beam_geometry(space, angles=None, det_shape=None):
+    """Create default parallel beam geometry from space.
+
+    This default geometry gives a fully sampled sinogram according to the
+    nyquist criterium. In general this gives a very large number of samples.
```

Comment:
Kak A C and Slaney M 2001 Principles of Computerized Tomographic Imaging (Philadelphia, PA: SIAM)